app/main/Playback/Playback.js

```diff
@@ -92,12 +92,15 @@
 
     /**
      * Preview the captured frames.
+     * 
+     * @param {Integer} number - Frame number that previewing begins
      */
-    previewCapturedFrames() {
+    previewCapturedFrames(number) {
+      console.log("Previewing from frame " + number);  
       // Display playback window
       if (global.projectInst.getCurrentMode() === "capture") {
         global.projectInst.setCurrentMode("playback");
-        this.curPlayFrame = 0;
+        this.curPlayFrame = number;
```

Comment:
Set the curPlayFrame in both capture and playback modes to fix the playback mode issue mentioned in the longer comment

Reply:
Although i'm not sure if this will break other app functionality. So will need to be tested

Reply:
Testing the above, I saw that setting curPlayFrame in both capture and playback mode it disables the ability to play from the selected frame when pressing the normal play button. If for example I have frame 13 selected and I click the play button, the entire animation is played backed and not the animation starting in frame 13 as it would have.

Reply:
I think a possible solution is to call the `_displayFrame(id)` function before calling `previewCapturedFrames()` when short play is activated. `_displayFrame` is the function called when a user selects frame from the framereel.

This should mean you won't need to add an id argument to `previewCapturedFrames()`. 

I must admit the playback code for Boats Animator is a bit strange!

Reply:
Also would you be able to make the shortPlay code a method in Playback.js? Then it can be reused when a keyboard shortcut or menu bar item is added

Reply:
> Also would you be able to make the shortPlay code a method in Playback.js? Then it can be reused when a keyboard shortcut or menu bar item is added

You mean write a different method instead of using `previewCapturedFrames()`?  (Since previewCapturedFrames is already defined in Playback.js)

Reply:
I mean change:

```javascript
btnShortPlay.addEventListener("click", function() {
  if (takeInst.getTotalFrames() > 0) {
          if (!global.projectInst.playback.isPlaying) {
            const number = takeInst.getTotalFrames() - 5; 
            global.projectInst.playback.previewCapturedFrames(number);
          } else {
            console.log("already playing...");
          }
        }
});
```

To:

```javascript
btnShortPlay.addEventListener("click", function() {
  global.projectInst.playback.shortPlay();
});
```

and then in Playback.js add:

```javascript
shortPlay() {
  if (takeInst.getTotalFrames() > 0) {
          if (!global.projectInst.playback.isPlaying) {
            const number = takeInst.getTotalFrames() - 5; 
            global.projectInst.playback.previewCapturedFrames(number);
          } else {
            console.log("already playing...");
          }
        }
}
```

(some variable names will probably need changing here)